.github/workflows/licensed.yml

```diff
@@ -0,0 +1,20 @@
+name: Licensed
+
+on:
+  push: {branches: main}
+  pull_request: {branches: main}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: Check licenses
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm ci
```

Comment:
This could be a little faster with some caching, as in https://github.com/actions/github-script/blob/fb0feb9324d07280c5f37dd77a9f45a809435c95/.github/workflows/licensed.yml#L15-L19.